components/descriptions/DescriptionsItem.razor.cs

```diff
@@ -34,12 +34,14 @@ protected override void Dispose(bool disposing)
         {
             this.Descriptions?.Items.Remove(this);
             base.Dispose(disposing);
+            this.Descriptions?.ReRender();
         }
```

Comment:
It‘s better to add a method for Descriptions naming 'AddItem' and RemoveItem, then call them in DescriptionItems.

Reply:
done